src/components/Trade/hooks/useSwapper/calculateAmounts.ts

```diff
@@ -17,20 +17,22 @@ export const calculateAmounts = async ({
   swapper: Swapper
   action: TradeAmountInputField
 }) => {
-  const sellAssetUsdRate = bnOrZero(
-    await swapper.getUsdRate({
-      symbol: sellAsset.symbol,
-      tokenId: sellAsset.tokenId,
-    }),
-  )
-  const buyAssetUsdRate = bnOrZero(
-    await swapper.getUsdRate({
-      symbol: buyAsset.symbol,
-      tokenId: buyAsset.tokenId,
-    }),
-  )
+  const sellAssetUsdRatePromise = swapper.getUsdRate({
+    symbol: sellAsset.symbol,
+    tokenId: sellAsset.tokenId,
+  })
 
-  const assetPriceRatio = buyAssetUsdRate.dividedBy(sellAssetUsdRate)
+  const buyAssetUsdRatePromise = swapper.getUsdRate({
```

Comment:
Since it's used with `Promise.all`, it's quite obvious they're `Promise`s
```suggestion
  const getBuyAssetUsdRate = swapper.getUsdRate({
```